File: mewsic-kmedia/src/commonMain/kotlin/org/mewsic/kmedia/container/mpeg4/MPEG4Container.kt
package org.mewsic.kmedia.container.mpeg4

import org.mewsic.kmedia.api.ContainerMedia
import org.mewsic.kmedia.api.ContainerMetadata

class MPEG4Container : ContainerMedia {
    override var metadata: ContainerMetadata = MpegMetadata()

}

File: mewsic-kmedia-rs/src/container/mpeg4/mpeg4_container.rs
use crate::api::{ContainerMedia, ContainerMetadata};
use crate::container::mpeg4::{Mpeg4BoxSignatures, Mpeg4Metadata};

pub struct Mpeg4Container {

}

impl Mpeg4Container {

}
impl ContainerMedia for Mpeg4Container {
    fn get_metadata(&self) -> Box<dyn ContainerMetadata> {

        Box::new(Mpeg4Metadata {})
    }
}